server/models/Profile.js

```diff
@@ -0,0 +1,101 @@
+import mongoose from 'mongoose'
+
+const profileSchema = mongoose.Schema({
+  profilePhoto: {
+    type: String,
+  },
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  gender: {
+    type: String,
+    enum: ['male', 'female', 'other'],
+  },
+  birthDate: {
+    type: Date,
+    required: true,
+  },
+  phoneNumber: {
+    type: String,
+  },
+  address: {
+    type: String,
+  },
+  description: {
+    type: String,
+  },
+  availability: [
+    {
+      day: {
+        type: String,
+        enum: ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday']
+      },
+      date: Date,
+      startTime: {
+        type: String,
+        enum: [
+          '1 am',
+          '2 am',
+          '3 am',
```

Comment:
This seems unnecessary, we can just have validation that determines if were inputting valid times into the form.